### DISCUSSION TITLE
Easy way to make all of private torrents seeded public?

### DISCUSSION THREAD

**Discussion by trancephorm** [rakshasa/rtorrent#1123] [Q&A]
So I have a huge collection of music seeded privately on one tracker. Is there a way that I can bulk insert addresses of public trackers too all the torrents? I guess if torrents are seeded on public trackers, clients should be able to find them by magnet links using DHT/PEX networks right?

Thanks for your attention :)

**Comment by rakshasa** [comment]
There are tools out there that you can use to edit `bencoded` files, which you can use to edit those files using a script.

Haven't looked at what is available lately, so do your own research.

**Comment by trancephorm** [comment]
Thanks! I found one.... so I just stop rtorrent and bulk edit them and it will be ok, no problem because I've changed torrents?

**Reply by rakshasa** [reply]
As long as you didn't change the `info` section, then yes it will work.

**Comment by CravateRouge** [comment]
@trancephorm Did you find a script or did you write one that you are willing to share?

**Reply by trancephorm** [reply]
I don't remember which one I used, but here's the one I just found: https://mkbrr.com/guides/modifying-torrents  In other news, AI coding is so powerful these days it's trivial to make such kind of utility yourself.
